src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java

```diff
@@ -61,7 +61,7 @@
     private static final String CASE_ID = "1000001";
     private static final String EMAIL_TEMPLATE_ID = "email-template-id";
     private static final String SMS_TEMPLATE_ID = "sms-template-id";
-    private static final String LETTER_TEMPLATE_ID_STRUCKOUT = "struckOut";
+    private static final String LETTER_TEMPLATE_ID_STRUCKOUT = null;
```

Comment:
Is this required?  If it is, I find it a little strange that it is set to `null`.  I don't think there are templateIds for bundled letters any more.